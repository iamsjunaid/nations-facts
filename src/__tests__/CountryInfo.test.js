import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Showcountry from '../components/CountryInfo';

describe('test if country info renders correctly', () => {
  test('country info snapshot', () => {
    const { container } = render(
      <Provider store={store}>
        <BrowserRouter>
          <Showcountry />
        </BrowserRouter>
      </Provider>,
    );
    expect(container).toMatchSnapshot();
  });
});
