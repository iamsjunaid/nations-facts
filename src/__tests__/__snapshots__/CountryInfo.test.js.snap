// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test if country info renders correctly country info snapshot 1`] = `
<div>
  <div>
    loading...
  </div>
</div>
`;
